src/components/EventForm/EventForm.tsx

```diff
@@ -66,7 +67,8 @@ function EventForm({ onSubmit, ...rest }: EventFormProps) {
   const [hasSubmitted, setHasSubmitted] = useState(false);
   const [error, setError] = useState<string | null>(null);
 
-  const isFormValid = name && description && eventType;
+  const isFormValid = name && description && eventType && startTime && endTime &&
+    new Date(date + 'T' + startTime) > new Date() && new Date(date + 'T' + endTime) > new Date();
```

Comment:
Could also use `parse` from `date-fns` instead of `new Date(x)`.

Reply:
Created a local function for it so the pattern isn't repeated so much